### PR TITLE
feat(notification): facebook workplace integration

### PR DESCRIPTION
**Pull request by adeyahya**
Hello everyone, I made this pr so that Monika can send notifications via Facebook Workplace. Please kindly check and review it thanks.

### CODE REVIEW

File: docs/src/pages/guides/notifications.md [hunk 1]
```diff
@@ -298,6 +299,27 @@ Monika supports Discord. To enable notification via Discord, you must create a d
...
+    "access_token": "your_custom_integration_access_token"
+  }
+}
+```
+
+| Key  | Description                                     | Example            |
+| ---- | ----------------------------------------------- | ------------------ |
+| ID   | Notification identity number                    | `Workplace12345`   |
+| Type | Notification types                              | `slack`            |
+| ID   | It's located at thread url, in the last segment | `6367478493277649` |
```
Comment (nicnocquee) [inline docs/src/pages/guides/notifications.md:321]: I think the key here should be `thread_id`, right? Can you also please add an example of the thread URL?

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Comment by dennypradipta** [comment]
Hello @adeyahya thank you for contributing!

May I ask you to provide screenshots of the notifications sent to Facebook Workplace? :grinning: 
You should probably censor the sensitive data if there is any.

Thank you

**Review by kevinhermawan — APPROVED** [review]
Looks good to me! Thanks for the PR @adeyahya 🍻

**Comment by adeyahya** [comment]
> Hello @adeyahya thank you for contributing!
> 
> May I ask you to provide screenshots of the notifications sent to Facebook Workplace? 😀
> You should probably censor the sensitive data if there is any.
> 
> Thank you

here is the screenshot
Screen Shot 2021-06-11 at 16 55 02

File: docs/src/pages/guides/notifications.md (hunk 1 above)
Comment (adeyahya) [reply docs/src/pages/guides/notifications.md:321]: ah ya, it should be thread_id, I will update it. and here is the thread URL example

https://workwork.workplace.com/chat/t/6367478493277649

File: docs/src/pages/guides/notifications.md [hunk 2]
```diff
@@ -298,6 +299,27 @@ Monika supports Discord. To enable notification via Discord, you must create a d
...
+    "thread_id": "12345678910",
+    "access_token": "your_custom_integration_access_token"
+  }
+}
+```
+
+| Key  | Description                                     | Example            |
+| ---- | ----------------------------------------------- | ------------------ |
+| ID   | Notification identity number                    | `Workplace12345`   |
+| Type | Notification types                              | `slack`            |
```
[the remark this answers is not in the document]
Comment (adeyahya) [reply docs/src/pages/guides/notifications.md]: omg 😆 updating, sorry

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**